python/salpytools/salpylib.py

```diff
@@ -107,25 +106,24 @@ def __init__(self, Device='atHeaderService', default_state='OFFLINE',
             self.load_state_enumeration()
         except Exception:
             LOGGER.warning("Cannot load state enumeration -- will use library instead")
-            self.detailedState_enum = states.state_enumeration
+            self.State_enum = states.state_enumeration
```

Comment:
I think this should be self.summaryState_enum to be explicit. Ditto for all the lines below that use this.

Reply:
Fixed